pkg/cloudevents/transport/http/transport.go

```diff
@@ -383,19 +383,26 @@ func (t *Transport) longPollStart(ctx context.Context) error {
 	req = req.WithContext(ctx)
 	msgCh := make(chan Message)
 	defer close(msgCh)
+	isClosed := false
 
 	go func(ch chan<- Message) {
 		for {
+			if isClosed {
+				return
+			}
+
 			if resp, err := t.LongPollClient.Do(req); err != nil {
 				logger.Errorw("long poll request returned error", err)
 				uErr := err.(*url.Error)
 				if uErr.Temporary() || uErr.Timeout() {
+					time.Sleep(time.Second * 2)
```

Comment:
not sure why you would want to add sleep statements here. Is this for debugging and forgotten?

Reply:
when get error here， it retry "DO" too many times. then you will get too many error log, so I think here should retry after some time.

Reply:
We can't add seep commands to the codebase like this.

Reply:
OK, I've deleted the sleep code.